src/main/java/by/dragonsurvivalteam/dragonsurvival/client/gui/SkinsScreen.java

```diff
@@ -50,7 +50,7 @@ public class SkinsScreen extends Screen{
 	private static final ResourceLocation WIKI = new ResourceLocation(DragonSurvivalMod.MODID, "textures/gui/wiki_button.png");
 	private static final ResourceLocation HELP = new ResourceLocation(DragonSurvivalMod.MODID, "textures/gui/skin_help.png");
 
-	private static final String DISCORD_URL = "http://discord.gg/8SsB8ar";
+	private static final String DISCORD_URL = "https://discord.gg/8SsB8ar";
```

Comment:
Is there any impact in terms of defensiveness here? The Discord is supposedly already protected

Reply:
saves you a redirect to the https url
![image](https://github.com/DragonSurvivalTeam/DragonSurvival/assets/22003703/ee90a71a-11a2-40ab-a1ef-b66789513163)